assets/src/blocks/Gallery/GalleryThreeColumns.js

```diff
@@ -1,26 +1,28 @@
 import {IMAGE_SIZES} from './imageSizes';
+import {GalleryLightbox} from './GalleryLightbox';
+import {GalleryImage} from './GalleryImage';
 
 const ordinals = ['first', 'second', 'third'];
 
-export const GalleryThreeColumns = ({images, postType, onImageClick}) => (
+export const GalleryThreeColumns = ({images, postType, expand}) => (
   <div className="three-column-images row">
     {images.slice(0, 3).map((image, index) => (
       <div className="col" key={image.image_src}>
         <div className={`${ordinals[index]}-column split-image`}>
-          {image.image_src &&
-            <img
-              loading="lazy"
-              src={image.image_src}
-              srcSet={image.image_srcset}
-              sizes={IMAGE_SIZES[`threeColumns${index}`]}
-              style={{objectPosition: image.focus_image}}
-              alt={image.alt_text}
-              title={image.alt_text}
-              className={`img_${postType}`}
-              data-index={index}
-              onClick={onImageClick}
-              role="presentation"
-            />
+          {
+            expand ?
```

Comment:
This code piece is repeated 3 times (one per style), maybe it could be refactored?